tests/sets/set_senior_never_alone_app.yaml

```diff
@@ -30,4 +30,5 @@ SetSeniorNeverAloneAppTest:
     - Case: TestSeniorSettingsPageNavigation
     - Case: TestSeniorWifiSettingsPage
     - Case: TestSeniorAppDeactivatedAccountStatusVisuals
-    - Case: TestSeniorIpadShouldNotResetIfCarePartnerUpdatesParticipantProfile
\ No newline at end of file
+    - Case: TestSeniorIpadShouldNotResetIfCarePartnerUpdatesParticipantProfile
+    - Case: TestSeniorAppVisualExperienceIfLoseConnectionOnCall
```

Comment:
As this is a physical device only test, only add it to the physical set. The ones divided by modules don't have physical tests.

Reply:
It 've removed.